src/scss/components/_controls.scss

```diff
@@ -7,7 +7,7 @@
     position: absolute;
     top: 0;
     left: 0;
-    z-index: 2;
+    z-index: 3;
```

Comment:
it's set at 3 because the of other z-index levels.  These could _probably_ be pulled out into variables so they can be more easily managed.